src/User/pages/Dashboard/ProfilePage.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react';
+import Aside from '../../components/Aside/Aside';
+
+const ProfilePage = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    mobile: '',
+    password: '',
+    profilePicture: null, // Add a state to store the profile picture
```

Comment:
add a sample avatar till then they do not add any image

Reply:
Yes @codervivek5 i will add